CalendarApp/NUnitTestProject/BookingServiceTests.DeleteTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3;
+using NUnit.Core;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Microsoft.Practices.Unity;
+using GoogleCalendarService;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        private IUnityContainer _unityContainer;
+        private CalendarService _calendarService;
+        private BookingService _meetingBooking;
+
+        [TestFixtureSetUp]
+        public void Initialize()
+        {
+            _unityContainer = Dependency.UnityConfig.GetUnityContainer();
+            _calendarService = _unityContainer.Resolve<CalendarService>();
+            _meetingBooking = _unityContainer.Resolve<BookingService>();
+        }
+
+        [Test]
+        public void BookingService_DeleteEvent_WrongCalendarIdThrowException()
+        {
+            var eventId = "abqq3960anfn5cmi1242usco";
+            var calendarId = string.Empty;//error
+
+            Assert.Throws(typeof(Google.GoogleApiException),
+               () => { _meetingBooking.DeleteEvent(eventId, calendarId); });
+        }
+
+        [Test]
+        public void BookingService_DeleteEvent_WrongEventIdThrowException()
+        {
+            var eventId = string.Empty;//error
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            Assert.Throws(typeof(Google.GoogleApiException),
+               () => { _meetingBooking.DeleteEvent(eventId, calendarId); });
+        }
+
+        [Test]
+        public void BookingService_DeleteEvent_DeletesEventCorrectly()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
```

Comment:
What is the reason of strongly typed class for application setting has not been created instead of `AppSettingsHelper`?